soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3981,6 +3998,15 @@ void RandomizerSettingsWindow::DrawElement() {
 
                 UIWidgets::PaddedSeparator();
 
+                // Shuffle Hyrule Loach Reward
+                UIWidgets::EnhancementCheckbox(Settings::ShuffleHyruleLoachReward.GetName().c_str(), "gRandomizeShuffleHyruleLoachReward");
+                UIWidgets::InsertHelpHoverText(
+                    "Shuffles a Purple Rupee into to the item pool, and allows\n"
```

Comment:
Manual line breaks aren't necessary.  Text is automatically wrapped.

Reply:
I added the line breaks because the "Shuffle Frog Song Rupees" option uses them.

Reply:
It's wrong too 😊 It's not like it hurts anything, but it's not necessary.  It's likely that it was just copied from the 3Drando setting descriptions file, which doesn't have automatic line breaking.

Reply:
Fixed it.